wg-meetings/2017-11-29.md

```diff
@@ -0,0 +1,97 @@
+       for others to contribute as well as make sure that model in the doc is understandable by others.    
+    - **ACTIONS**:
+      - [ ] @mike-kaufman - figure out how to grant access to early draft of md
+      - [ ] @mrkmarron, @mike-kaufman - get draft polished up enough to make public & make a PR to diagnostics GH repro
+
+  - **Trace Events** tracking issue #84
+     - @ofrobots, @kjin are ramping up on the space
+     - Haven't updated trackign issue
+     - Stand-alone library factored out of chrome hasn't been done yet.
+     - There's an issue currently in how to keep the trace macros in sync.
+         - One copy lives under `deps/v8/`
+         - another copy under `src/` that is effectively a copy of above w/ a few slight mods
+         - currently, this update needs to happen w/ each rev of v8
+         - it's annoying 
+
+  - **Integrate C++ AsyncHooks Embedder API with native abstraction** [#13254](https://github.com/nodejs/node/issues/13254)
+    - Done for Napi
+    - Not done for NaN
+    - How do we move this forward?
+      - @andreasMadsen would like to sync w/ NaN people & talk to them about the details
+      - open ? on who the right people are w/ deep enough understanding of NaN
+      - @AndreasMadsen hasn't gotten a lot of traction with ppl he's contacted
+    - Question is how high of a priority is this?
+      - Issue in that anyone using a NaN native module & AsyncHooks is at risk of losing context
+        - e.g., users of native redis module
+      - @jkrems - how many modules use NaN APIs now? 
+      - @ofrobots - issue of correctness, that is, w/out having NaN updated, AsyncHooks is effectively unreliable & can't be counted on
+    - **ACTIONS:**  
+      - [ ] @mike-kaufman - add trackign issue for NaN support to AsyncHooks tracking issue if not already present 
+
+  - **[async_hooks] stable API - tracking issue** [#124](https://github.com/nodejs/diagnostics/issues/124)
+    - How do we get this across finish line?
+    - Trevor no longer involved, Anna has enough context to help out
+    - Issues exist now where we have context, but it is wrong context
+    - We do have some users: groupon, new relic, Google's stack drvier
+    - Need to get feedback on quality level of async hooks from users, ideally users at scale who have broad insight into how well it is working
+    - in particular, are people having to monkey-patch around async-hooks losing context?
+    - **ACTIONS:**
+      - [ ] @mike-kaufman - open issue to request feedback from current users, /cc active maintainers 
+
+  - **Diagnostics WG Summit** [#121](https://github.com/nodejs/diagnostics/issues/121)
+    - looks like 2/12 & 2/13 are the dates
+    - Should confirm end of week (12/1)
+    - **ACTIONS:**
+      - [ ] @mhdawson - propose detailed schedule
+
+  - **Perf/Node.js** [#168](https://github.com/nodejs/benchmarking/issues/168)
+    - @mike-kaufman had to leave for this part & missed discussion here.

```

Comment:
- perf need to be updated to understand interpreter
- @hashseed will progress on V8 side to see if this is possible
- lastest info https://github.com/nodejs/benchmarking/issues/168#issuecomment-348434328

Reply:
👍